### PR TITLE
Added Recipe and RecipeView components for home page

### PR DESCRIPTION
**Pull request by Lord-Bubbles**
#16

### CODE REVIEW

**Comment by Lord-Bubbles** [comment]
Back end is not implemented and style is still not perfect but here are pictures of what it looks like with manually-inputted dummy data:
https://drive.google.com/file/d/1LlQNr5T03p5lbEmFfsiWaJcZfuqJATRD/view?usp=sharing
https://drive.google.com/file/d/1KOgSUm5OWaSSAQV4B6dfBb2yaqKlx7ju/view?usp=sharing
https://drive.google.com/file/d/1LivQJPxGLJJHTkIvTdgxWcI34EqCdVmO/view?usp=sharing

File: src/components/Recipe.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      modal: false,
+    };
+  }
+
+  setModal = (flag) => {
+    this.setState({modal: flag});
+  }
+
+  displayIngredients = () => {
+    let ingredList = [];
```
Comment (chilimalone) [inline src/components/Recipe.js:18]: nit: spell it all out: ingredientList

File: src/components/Recipe.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        <div className="rounded-border d-inline p-1 bg-light">
+          <Card border="dark">
+            <Card.Img variant="top" src=""/>
+            <Card.Title>
+              <button className="card-title" onClick={() => this.setModal(true)}>
+                <strong>{this.props.data.name}</strong>
+              </button>
+            </Card.Title>
+            <Card.Body>
+              <div>Time: {this.props.data.time} min</div>
```
Comment (chilimalone) [inline src/components/Recipe.js:48]: Is your data.time always in minutes? What if it's 4 hours? Will it say 240 min instead?

File: src/components/Recipe.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          onHide={() => this.setModal(false)}
+          centered="true"
+          size="lg"
+        >
+          <Modal.Header closeButton>
+              <Modal.Title>{this.props.data.name}</Modal.Title>
+              <Modal.Title className="modalTitle">{this.props.data.time} min</Modal.Title>
+          </Modal.Header>
+          <img 
+              src=""
```
Comment (chilimalone) [inline src/components/Recipe.js:64]: empty image?

File: src/styles/Recipe.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  outline: none;
+}
+
+.modal-box {
+  border-style: ridge;
+  outline: black;
+  background-color: transparent;
+}
+
+.modalTitle {
```
Comment (chilimalone) [inline src/styles/Recipe.css:15]: nit: be consistent with your naming. everything else is -notation, so here should be modal-title

File: src/App.js [hunk 5]
```diff
@@ -29,7 +35,7 @@ class App extends React.Component {
     document.getElementById("signInDiv").hidden = false;
   }
 
-  componentDidMount() {
+  async componentDidMount() {
```
Comment (chilimalone) [inline src/App.js:38]: i don't think this needs async? you're not calling await on anything

**Pull request closed without merging**